backend/controller/observability/observability.go

```diff
@@ -2,10 +2,22 @@ package observability
 
 import "fmt"
 
+var (
+	fsm *FSMMetrics
+)
+
 func InitControllerObservability() error {
-	if err := InitFSMMetrics(); err != nil {
+	var err error
+
+	fsm, err = InitFSMMetrics()
+
+	if err != nil {
 		return fmt.Errorf("could not initialize controller metrics: %w", err)
 	}
 
 	return nil
 }
+
+func FSM() *FSMMetrics {
+	return fsm
+}
```

Comment:
We can probably just expose the public var here vs. having a wrapper func. This is likely more true with an `init()` approach.

Reply:
feedback adopted